### DISCUSSION TITLE
DAX Date Template vs. Bravo Date Table

### DISCUSSION THREAD

**Discussion by ericdomond** [sql-bi/Bravo#422] [Q&A] [answered]
Because Bravo saves _so much time_ by generating Time Intelligence variants, I want to leverage it immediately. However, Bravo overwrites pre-existing date tables, so I have to choose between the original robust [DAX Date Template](https://www.sqlbi.com/tools/dax-date-template/), and the more streamlined options available in Bravo. (I have modified the date template's holidays to include regional holidays that are missing from my country's holidays in the original code, so these are also overridden.)

Are there plans to add columns to the tables that Bravo generates to make it as robust as the original date table template? Or plans to stop Bravo from modifying a pre-existing date table but still generating time intelligence variants? (I realize this will be harder, since the variants depend on that date table.)

**Comment by marcosqlbi** [comment, marked as answer]
We are working on a system to provide customized templates for date tables and time intelligence functions.
The plan is to release such a feature for v1.0.

**Comment by marcosqlbi** [comment]
@ericdomond please try version 0.9.6 - it has the feature to customize the date templates!
We are still working on documentation, which is a work in progress.
Check this page and related links:
https://docs.sqlbi.com/bravo/customize-date-template
